allauth/account/views.py

```diff
@@ -654,6 +654,12 @@ def logout(self):
                                   messages.SUCCESS,
                                   'account/messages/logged_out.txt')
         auth_logout(self.request)
+        signal_kwargs = {}
+        signals.user_logged_out.send(sender=self.request.user.__class__,
+                                    request=self.request,
+                                    user=self.request.user,
```

Comment:
I tend to prefer keeping signals simple, so preferably no pre/post logged out versions. If you need that kind of control, you can already override the adapter and insert pre/post logic as you wish. So, the only thing that needs to be done here is firing a `user_logged_out` signal like so (pseudo code):

```
user = self.request.user
logout()
signals.user_logged_out.send(...user=user)
```